src/traces/parcoords/attributes.js

```diff
@@ -141,6 +141,21 @@ module.exports = {
         description: 'The dimensions (variables) of the parallel coordinates chart. 2..60 dimensions are supported.'
     }),
 
+    bgline: {
+        color: {
```

Comment:
If we follow the example of `scatter` we get `unselected.line.color`. It seems a bit overly-nested, though `splom` does this too and all it has in `unselected` is `marker`. I think we should do it, gives us maximum extensibility as well as matching other trace types.

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/4635/commits/9b4711b74cd5c30dfccdce3ef27b875c2856f65a.